Orthtree/doc/Orthtree/Orthtree.txt

```diff
@@ -5,7 +5,7 @@ namespace CGAL {
 \anchor Chapter_Orthtree
 
 \cgalAutoToc
-\authors Jackson Campolattaro, Simon Giraudot, Cédric Portaneri, Tong Zhao, Pierre Alliez
+\authors Jackson Campolattaro, Simon Giraudot, Cedric Portaneri, Tong Zhao, Pierre Alliez
```

Comment:
I do not get that. Why remove the accent from Cédric's name?!

Reply:
Ah, that was me. Visual Studio didn't showed the accent, but some weird characters. I'll fix it.